storage/store.go

```diff
@@ -1213,6 +1223,10 @@ func (s *Store) ExecuteCmd(ctx context.Context, args proto.Request) (proto.Respo
 			// This error needs to be converted appropriately so that
 			// clients will retry.
 			err = proto.NewRangeNotFoundError(rng.Desc().RangeID)
+		} else if iErr, ok := err.(*errWithIndex); ok {
```

Comment:
do you mean `unwrapIndexedError`? That doesn't give us the index (and even if it did, it's a test-only helper that's probably best kept there given that this line here will be the only explicit unwrap in production code).